templates/web/brent/waste/services.html

```diff
@@ -63,7 +63,7 @@
   </form>
   [% END %]
   [% IF NOT pending_cancellation %]
-      [% IF NOT unit.garden_due AND NOT waste_features.garden_modify_disabled %]
+      [% IF NOT unit.garden_due AND NOT waste_features.garden_modify_disabled AND is_staff %]
```

Comment:
You've put this one back but previously there was another one on line 33 as well, presumably should also be there to match?

Reply:
Seems a bit deliberate but I don't see that the conversation changed from this being staff only